documentation/general/design/dash-holistic-view.md

```diff
@@ -0,0 +1,639 @@
+---
+Note: Work in progress
+Last update: 04/26/2022
+---
+
+[<< Back to parent directory](../README.md) ]
+
+[<< Back to DASH top-level Documents](../../README.md#contents) ]
+
+# Disaggregated API for SONiC Hosts (DASH) high level design 
+
+- [Introduction](#introduction)
+  - [Why DASH?](#why-dash)
+  - [Objectives](#objectives)
+- [Compliance and requirements](#compliance-and-requirements)
+- [Scenarios](#scenarios)
+- [Azure provisioning example](#azure-provisioning-example)
+- [DASH project](#dash-project)
+- [Logical architecture (software stack)](#logical-architecture-software-stack)
+    - [SDN controller](#sdn-controller)
+      - [SDN and DPU High-Availability (HA)](#sdn-and-dpu-high-availability-ha)
+  - [gNMI container](#gnmi-container)
+    - [Multiple DPUs device](#multiple-dpus-device)
+  - [SONiC app containers](#sonic-app-containers)
+  - [Switch State Service (SWSS)](#switch-state-service-swss)
+  - [Switch Abstraction Interface (SAI) DASH extension](#switch-abstraction-interface-sai-dash-extension)
+  - [ASIC Drivers](#asic-drivers)
+  - [DASH capable ASICs](#dash-capable-asics)
+- [SONiC integration](#sonic-integration)
+  - [DASH single DPU on NIC](#dash-single-dpu-on-nic)
+  - [DASH appliance](#dash-appliance)
+    - [High Level Architecture](#high-level-architecture)
+    - [Low level architecture](#low-level-architecture)
+  - [DASH smart switch](#dash-smart-switch)
+    - [High level architecture](#high-level-architecture-1)
+    - [Low level architecture](#low-level-architecture-1)
+- [Example of network representation](#example-of-network-representation)
+  - [Deployment](#deployment)
+- [API](#api)
+  - [Switch Abstraction Interface (SAI) API](#switch-abstraction-interface-sai-api)
+  - [DASH pipeline API](#dash-pipeline-api)
+    - [Behavioral model](#behavioral-model)
+  - [Functional testing](#functional-testing)
+  - [Conformance and performance testing](#conformance-and-performance-testing)
+- [Appendix](#appendix)
+  - [A day in the life of a DASH packet](#a-day-in-the-life-of-a-dash-packet)
+  - [A day in the life of a DASH SDN controller](#a-day-in-the-life-of-a-dash-sdn-controller)
+  - [A day in the life of a DASH container](#a-day-in-the-life-of-a-dash-container)
+- [References](#references)
+
+## Introduction
+
+> [!NOTE]
+> This document is work in progress and will replace the current [SONiC-DASH High Level Design (WIP)](dash-high-level-design.md).
```

Comment:
Please keep this as the original file name. we've few references to this file.

Reply:
Done